public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -565,14 +489,13 @@ async function Sidebar() {
 
    
 </div>`,
-			"usage": `<div id="sb4"  class="custom_sidebar">
-    <div class="custom_field">
-         <a onclick = "event.preventDefault(); buildcite()" href='#' class='mt-icon-quote'>&nbsp;Get Page Citation</a>
-    </div>
+            "usage": `<div id="sb4"  class="custom_sidebar">
+	<div class="custom_field">
+		<a onclick = "event.preventDefault(); saveBookmark()" href='#' class='mt-icon-quote'>&nbsp;Save Bookmark</a>
+			<div id="bm-list">
 
-    <div class="custom_field">
-         <a onclick = "event.preventDefault(); attribution()" href='#' class='mt-icon-quote'>&nbsp;Get Page Attribution</a>
-    </div>
+			</div>
+	</div>
```

Comment:
It looks like get citation and get attribution have been removed. Is this intentional?

Reply:
Yes